src/components/carousel/types.ts

```diff
@@ -66,6 +66,13 @@ export interface CarouselProps extends ScrollViewProps {
    * whether to show a page counter (will not work with 'pageWidth' prop)
    */
   showCounter?: boolean;
+  /**
+   * the counter's style
+   */
+  /**
+   * the counter's container style
+   */
+ counterContainerStyle?: StyleProp<ViewStyle>;
```

Comment:
@alwaysintune @adids1221
Generally, we should avoid adding styles for each view in the hierarchy.
Let's try to find a solution that solves the issue without an additional prop.
Please open a ticket if needed.